cmd/runtimetest/main.go

```diff
@@ -778,6 +779,20 @@ func validatePosixMounts(spec *rspec.Spec) error {
 	return mountErrs
 }
 
+func validateMountLabel(spec *rspec.Spec) error {
+	for _, mount := range spec.Mounts {
+		fileLabel, err := label.GetFileLabel(mount.Destination)
+		if err != nil {
+			return fmt.Errorf("Failed to get mountLabel of %v", mount.Destination)
+		}
+		if fileLabel != spec.Linux.MountLabel {
```

Comment:
Linux judgment should be increased to avoid panic.

Reply:
updated